src/dmd/dsymbol.d

```diff
@@ -585,7 +585,7 @@ extern (C++) class Dsymbol : RootObject
         return s;
     }
 
-    const(char)* kind() const
+    const(char)* kind() const pure nothrow @nogc @safe
```

Comment:
Are the c++ headers marked as `const() const` too?

Reply:
This PR does not change the `const` attributes on `kind()`.